ibm/service/kubernetes/resource_ibm_container_vpc_cluster.go

```diff
@@ -184,56 +187,62 @@ func ResourceIBMContainerVpcCluster() *schema.Resource {
 			},
 
 			"worker_count": {
-				Type:        schema.TypeInt,
-				Optional:    true,
-				Default:     1,
-				Description: "Number of worker nodes in the cluster",
+				Type:             schema.TypeInt,
+				Optional:         true,
+				Default:          1,
+				DiffSuppressFunc: flex.ApplyOnce,
+				Description:      "Number of worker nodes in the cluster",
```

Comment:
it only sets the worker count for the default pool, right? maybe something like this?
```suggestion
				Description:      "Number of worker nodes in the default worker pool",
```